Cargo.toml

```diff
@@ -144,6 +144,20 @@ winit = "0.29"
 zip = "0.6"
 prettify-js = "0.1.0"
 indexmap = { version = "2", features = ["serde"] }
+argon2 = "0.5.2"
```

Comment:
Do not use precise versioning unless you really care. Prefer `0.5` over `0.5` and `1` over `1.3.5` etc.